modules/compiler/targets/host/source/target.cpp

```diff
@@ -73,6 +73,14 @@ static llvm::TargetMachine *createTargetMachine(llvm::StringRef CPU,
     Options.MCOptions.ABIName = ABI;
   }
 
+  std::optional<llvm::CodeModel::Model> CM;
+  // TODO: RISC-V (currently) needs the 'medium' code model as we don't ensure
```

Comment:
what's the TODO saying - that we should switch to small and support it correctly or just investigate if this is correct?

Reply:
Maybe we can remove the `TODO` as it's not clear (to me, at least) whether this is a bug that needs to be fixed or an acceptable convention we are okay with in the RISC-V host target.

Reply:
that makes sense